dub.sdl

```diff
@@ -67,7 +67,11 @@ configuration "windows-win32" {
   dependency ":base" version="*"
 }
 
-# Test configurations
+# unittest settings
+buildType "optimised-unittest" {
```

Comment:
BTW, I prefer to use American English, i.e. `optimized`. It would also otherwise be inconsistent with the Dub build option which is called `optimize`.

Reply:
No worries, I've changed it.